app/org/sagebionetworks/bridge/dao/ScheduledActivityDao.java

```diff
@@ -2,6 +2,7 @@
 
 import java.util.List;
 
+import org.joda.time.Chronology;
```

Comment:
I don't think you need Chronology, at least elsewhere I've called dateTime.getZone() without issue.